src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -3,12 +3,54 @@
 
 #pragma once
 
+#include <cstddef>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Domain/Creators/DomainCreator.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/OptionTags.hpp"
+#include "Domain/Structure/ExcisionSphere.hpp"
 #include "Options/Options.hpp"
 
 namespace CurvedScalarWave::Worldtube {
 namespace OptionTags {
 struct Worldtube {
   static constexpr Options::String help = {"Options for the Worldtube"};
 };
+
+struct ExcisionSphere {
+  using type = std::string;
+  static constexpr Options::String help{
+      "The name of the excision sphere as returned by the domain."};
+  using group = Worldtube;
+};
+
 }  // namespace OptionTags
+
+namespace Tags {
+template <size_t Dim>
+struct WorldtubeRadius : db::SimpleTag {
+  using type = double;
+  using option_tags = tmpl::list<domain::OptionTags::DomainCreator<Dim>,
+                                 OptionTags::ExcisionSphere>;
+  static constexpr bool pass_metavariables = false;
+  static double create_from_options(
+      const std::unique_ptr<::DomainCreator<Dim>>& domain_creator,
```

Comment:
I think the argument type should be `const DomainCreator<Dim>&`. This is easy to test with `TestHelpers::test_option_tag_factory_creation`. Please use that in your test.

Reply:
no, I think it's a pointer. I don't understand, `TestHelpers::test_option_tag_factory_creation` tests option tags with the type of a base class pointer which I do not have here.

Reply:
You're right. I don't remember if we have a quick way to test an option-created tag like this. The way you did seems fine.